src/prebid.js

```diff
@@ -110,6 +120,61 @@ function init(timeout, adUnitCodeArr) {
 	}
 }
 
+/*
+ *	get Sizes by viewable area size
+ */
+function getSizeByDynamicSizes(dynamicSizes){
+	var size = [];
+	var defaultSize =[];
+
+	//get viewable area size
+	var wWidth = utils.getViewableWidth();
+	var wheight = utils.getViewableHeight();
+
+	for( var i=0; i<dynamicSizes.length; i++){
+
+		var first = dynamicSizes[i][0];
+
+		//default size [0,0]
+		if(first[0] ===0 && first[1] ===0){
```

Comment:
this code isn't safe to execute because it's taking input from external sources. You need to validate it first